src/services/recipe.service.ts

```diff
@@ -73,10 +71,20 @@ export class RecipeService {
     return null;
   }
 
+  public async deleteRecipeImages(recipeId: string): Promise<boolean> {
+    const count = await Image.countDocuments({ recipeId });
+    const deleted = await Image.deleteMany({ recipeId });
+    const deletedCount = deleted?.deletedCount;
+    if (count === deletedCount) {
```

Comment:
Can you explain this if statement? I don't get why it's used here, maybe I don't get how `deleteMany` works

Reply:
@arkus7 `deleteMany` deletes items from database using optional query (in this case `{ recipeId }`) and returns object containing ok, n and deletedCount.

- n and deletedCount is the same, so I used the variable that is more descriptive.
- TypeScript didn't let me to write sth like this:

`const deletedCount = await Image.deleteMany({ recipeId })?.deletedCount;`
`if (count === deletedCount) {`

Reply:
If u wrap promise in parentheses u can do this in one line
`const deletedCount = (await Image.deleteMany({ recipeId }))?.deletedCount`

also if else block could be replaced with
`return count === deletedCount;`